src/components/Share/index.js

```diff
@@ -1,41 +1,62 @@
 import React, { Component, PropTypes } from 'react';
-const Style = require('./style.scss');
-export default class Share extends Component {
-
+import Style from './style.scss';
 
+export default class Share extends Component {
   static propTypes = {
     surah: PropTypes.object.isRequired
   };
 
+  getId() {
+    const { surahId } = this.props.surah;
+    return surahId;
+  }
+
+  getName() {
+    const { _, name } = this.props.surah;
+    return name;
+  }
+
+  getNameSimple() {
```

Comment:
I did think of that but I believe it is better convention if we have variant as a suffix instead of prefix easy to spot that its a variant of name IMO